### PR TITLE
balloons: add MaxBalloons configuration option

### PR DESCRIPTION
**Pull request by askervin**

### CODE REVIEW

File: pkg/cri/resource-manager/policy/builtin/balloons/balloons-policy.go [hunk 1]
```diff
@@ -935,13 +942,14 @@ func (p *balloons) setConfig(bpoptions *BalloonsOptions) error {
 		return err
 	}
 	balloons = append(balloons, defaultBalloon)
+	p.balloons = balloons
```
Comment (jukkar) [inline pkg/cri/resource-manager/policy/builtin/balloons/balloons-policy.go:945]: Very minor nit, do we need the `balloons` variable after this, could we get rid of it?

Comment (askervin) [reply pkg/cri/resource-manager/policy/builtin/balloons/balloons-policy.go]: Thanks @jukkar! Fixed.

**Review by jukkar — APPROVED** [review]
LGTM

**Review by klihub — APPROVED** [review]
LGTM

**Pull request merged**